cli/cli.ts

```diff
@@ -2047,7 +2047,6 @@ async function buildSemanticUIAsync(parsed?: commandParser.ParsedCommand) {
         generateReactCommonCss("skillmap"),
         generateReactCommonCss("authcode"),
         generateReactCommonCss("multiplayer"),
-        generateReactCommonCss("teachertool")
```

Comment:
I'm surprised. Did something change, or has this always been unnecessary?

Reply:
It's always been unnecessary, and was causing the react-common css to be loaded twice. Duplicates of every class.